### ISSUE TITLE
Improve accessibility of the default templates

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/news-vj-sharetools#8]
https://github.com/bbc/news-vj-sharetools/blob/master/bin/templates/dropdown/template.tmpl#L2-L5

Catharine Green's recommendation:

In the overlay, <p>Share</p> should be a button, to indicate in page activity. Also the close button has no label, let’s add some off-screen text that says ‘Close share’. I also see tabindex=”4” on the close button, is this necessary? The close button should also be marked up as a button element, not a link...

**Comment by connos** [comment]
I'm finding that I can't trigger the action on the share overlay links (Email, Facebook, or Twitter) using the keyboard. We should probably add that to the list of things to improve.

**Comment by ChrisBAshton** [comment]
Really, [these ones](https://github.com/bbc/news-vj-sharetools/blob/master/bin/templates/dropdown/template.tmpl#L11)? I'm surprised by that, as they're only links...

**Comment by connos** [comment]
I noticed they are only links. It surprised me too. Do they work for you?

I was expecting them to work when I hit the enter key.

**Comment by connos** [comment]
I can't even tab to them actually. ~~It might be to do with the tabindex attribute on the close buttons.~~ If I give the links tabindex="4" then I can tab to them and the enter key triggers the expected behaviour.

Removing the tabindex attribute from the close button (in the Chrome console) does not fix this problem, but we should definitely get rid of that attribute if it's not absolutely necessary.

**Comment by connos** [comment]
I've made the changes manually in the output of a local copy of the Salon project and I can tab properly now, at least in desktop Chrome. I've therefore changed the template in git and raised a PR. We should maybe discuss some proper testing.

**Comment by ChrisBAshton** [comment]
Fixed from 2.1.2
